jpegenc/setup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+setup(
+    name='JPEG Encoder',
+    version='0.1dev',
+    author='Merkourios Katsimpris',
+    author_email='merkourioskatsimpris@gmail.com',
+    url='https://github.com/mkatsimpris/test_jpeg',
```

Comment:
This looks good, we will want to merge this into `cfelton/test_jpeg` so we can collaborate per the emails.  The url should point to the original repo for now.

```
url=https://github.com/cfelton/test_jpeg
```
I am not sure how the author typically work, if it should list the three of us or single author is fine.

In addition to merging to your `master` these changes can pushed to original repo.